client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/launchpad-site-preview.tsx

```diff
@@ -29,7 +30,9 @@ const LaunchpadSitePreview = ( {
 	const isInVideoPressFlow = isVideoPressFlow( flow );
 
 	let previewUrl = siteSlug ? 'https://' + siteSlug : null;
-	const devicesToShow: Device[] = [ DEVICE_TYPES.COMPUTER, DEVICE_TYPES.PHONE ];
+	const devicesToShow: Device[] = isBlogOnboardingFlow( flow )
+		? [ DEVICE_TYPES.COMPUTER, DEVICE_TYPES.TABLET, DEVICE_TYPES.PHONE ]
+		: [ DEVICE_TYPES.COMPUTER, DEVICE_TYPES.PHONE ];
```

Comment:
> The tablet preview should only be visible for blog onboarding.

Out of curiosity, will that be a problem releasing it for other flows? It seems valuable having it there as well.

BTW it's not showing at setup/link-in-bio flow.

Reply:
I was being cautious since I hadn't considered the implications of including it in the other flows. On the face of it, though, it probably wouldn't be a problem. 🤔 How do you see it?

Reply:
Yeah, I would suggest releasing it for everyone since seems useful and safe 😃

<img width="1447" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/db6a3a41-ccd9-41d9-b3ea-d1ae95d28659">

Reply:
Ok. Let's go for it. ☺️